site/_docs/reference.md

```diff
@@ -2005,6 +2006,7 @@ Note:
 | JSON_TYPE(value)                                  | Returns a string indicating the type of a JSON **value**. This can be an object, an array, or a scalar type
 | JSON_DEPTH(value)                                 | Returns a integer indicating the depth of a JSON **value**. This can be an object, an array, or a scalar type
 | JSON_PRETTY(value)                                | Returns a pretty-printing of JSON **value**.
+| JSON_LENGTH(value)                                | Returns a integer indicating the length of a JSON **value**. This can be an object, an array, or a scalar type
```

Comment:
I would like to say the documentation MySQL specific operators should be improved somehow. The words `This can be an object, an array, or a scalar type` should only address the usage of JSON_TYPE but not JSON_DEPTH or JSON_LENGTH. And the parameters' datatype should be documented too.

But this is not a blocker, I can help with the document issue too after the patch get merged.